src/index.js

```diff
@@ -384,7 +384,12 @@ class JanusAdapter {
 
   configureSubscriberSdp(originalSdp) {
     if (!isH264VideoSupported) {
-      return originalSdp;
+      if (navigator.userAgent.indexOf("HeadlessChrome") !== -1) {
+        // HeadlessChrome (e.g. puppeteer) doesn't support webrtc video streams, so we remove those lines from the SDP.
```

Comment:
is Puppeteer being run in a CI environment (i.e., Travis) or on a developer's local machine?

if you invoke Puppeteer with `{ headless: false, args: "--enable-usermedia-screen-capturing" }`, does that work? (I've noticed that if you don't set `{ headless: false }`, then Puppeteer automatically adds the [`--disable-gpu` flag](https://github.com/GoogleChrome/puppeteer/blob/8fce319/lib/Launcher.js#L91) when Chromium is launched.)

related:
- https://bugs.chromium.org/p/chromium/issues/detail?id=737678
- https://github.com/GoogleChrome/puppeteer#q-what-features-does-puppeteer-not-support
- https://github.com/GoogleChrome/puppeteer/issues/2422#issuecomment-384752859

other than GoogleChrome/puppeteer#478, do you know if there is a Puppeteer issue filed for this?

Reply:
`headless: false` does not fix this. It's related to https://github.com/GoogleChrome/puppeteer/issues/2422#issuecomment-384752859 in that RTCPeerConnection will not accept h264 streams since puppeteer is a Chromium, not a Chrome.

Here's the error I get:
`OperationError: Failed to set remote offer sdp: Session error code: ERROR_CONTENT. Session error description: Failed to set remote video description send parameters`